addons/interact_menu/functions/fnc_addActionToClass.sqf

```diff
@@ -8,20 +8,37 @@
  * 1: Type of action, 0 for actions, 1 for self-actions <NUMBER>
  * 2: Parent path of the new action <ARRAY>
  * 3: Action <ARRAY>
+ * 4: Use Inheritance (Default: False) <BOOL><OPTIONAL>
```

Comment:
`<OPTIONAL>` looks strange
`(optional, default: false) <BOOL(EAN)>` ?
Maybe it's just me.